repository/src/main/java/org/apache/atlas/repository/store/graph/v2/preprocessor/sql/QueryCollectionPreProcessor.java

```diff
@@ -73,9 +73,7 @@ private void processCreate(AtlasStruct entity) {
     }
 
     private void processUpdate(AtlasStruct entity, AtlasVertex vertex) {
-        String vertexQnName = vertex.getProperty(QUALIFIED_NAME, String.class);
```

Comment:
I did not get reason to remove this logic

Reply:
Yeah, shouldn't be. Reverted change